Documentation/configuration.md

```diff
@@ -13,6 +13,7 @@ To start etcd automatically using custom settings at startup in Linux, using a [
 ##### -name
 + Human-readable name for this member.
 + default: "default"
++ This is the key used in the initial-cluster list (Ex: `default=http://localhost:2380`)
```

Comment:
I'd use the flag name here just to be _super_ clear:
...in the `-initial-cluster` list